### PR TITLE
[Compute] New command az vm assess-patches

### PR DESCRIPTION
**Pull request by qwordy**
**Description**  

Assess patches on the VM

**Testing Guide**  

```
az vm assess-patches -g rg -n vm
```

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yungezz** [comment]
hi @arrownj, could you pls review this PR? thanks

File: src/azure-cli/azure/cli/command_modules/vm/commands.py [hunk 1]
```diff
@@ -288,6 +288,7 @@ def load_command_table(self, _):
         g.generic_update_command('update', setter_name='update_vm', setter_type=compute_custom, supports_no_wait=True)
         g.wait_command('wait', getter_name='get_instance_view', getter_type=compute_custom)
         g.custom_command('auto-shutdown', 'auto_shutdown_vm')
+        g.command('assess-patch', 'assess_patches')
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/vm/commands.py:291]: minor thing, why patch in argument, patches in function?

Comment (yungezz) [reply src/azure-cli/azure/cli/command_modules/vm/commands.py:291]: btw, custom file missed in commit?

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/vm/commands.py]: Users don't want to type two more letters. assess_patches is a name defined in swagger.
No custom logic needed.

Comment (yungezz) [reply src/azure-cli/azure/cli/command_modules/vm/commands.py]: it's a minor thing. while it's better to align with swagger. user could get the hint from name it's plurality

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/vm/commands.py]: OK

**Pull request merged**